specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -388,7 +457,7 @@
         "clientSecretUrl": {
           "type": "string",
           "readOnly": true,
-          "description": " The ManagedServiceIdentity DataPlane URL that can be queried to obtain the identity credentials."
+          "description": " The ManagedServiceIdentity DataPlane URL that can be queried to obtain the identity credentials. If identity is user assigned, then the clientSecretUrl will not be present in the response, otherwise it will be present."
```

Comment:
For the 2015-08-31-PREVIEW API, the URL is returned for user assigned identities. Does it make sense to separate `IdentityProperties` into `SystemAssignedIdentityProperties` and `UserAssignedIdentityProperties`?

Reply:
Hum yeah it's probably a good idea.